### DISCUSSION TITLE
Repeatedly getting the same notification of the latest tag of ImagePolicy

### DISCUSSION THREAD

**Discussion by Revolution1** [fluxcd/flux2#2734] [Q&A] [answered]
environment:
- latest flux2
- discord provider
- ECR image registry with image policy and automation
- app are deployed with helmrelease

I setup a provider, and an alert for ImagePolicy.
Then I found, every 5 minute, I see a notification popped up telling me what is the latest image tag.
But the image repository had never changed during that time.

I checked kubernetes, the event did created every 5min

```
0s          Normal   info               imagepolicy/webui                 Latest image tag for 'xxxx/xxxx' resolved to: main-5e36d23e-32
```

It's there any way to not creating new event when nothing changes
or filter out the repeated events?

**Comment by kingdonb** [comment, marked as answer]
I have removed `ImagePolicy` resources from my notifications/Alerts for this very reason.

I'm sure this is planned to be addressed soon. We generally intend for notifications to only be raised when something has changed. We are aware that `ImagePolicy` do not behave this way.

You should still be able to get all or most of the important information by following the notifications from `ImageRepository`, `ImageUpdateAutomation`, `GitRepository`, `Kustomization`, or some subset of those resources.

So for now, the suggestion is to simply do not include `ImagePolicy` in your `Alerts` resources, but this is on the radar to be fixed.
